### PR TITLE
Added support for requesting motorcycles, mopeds/scooters 

### PR DESCRIPTION
**Pull request by Calimerorulez**
# Proposed Changes

Made APK date optional and added "N.v.t." as valid value for Interior because two wheeled vehicles do not have those :)

Mabye it is over the top, so you have to decide of you like it to be added or not. I personally find it useful :)

/edit: I see I've made a mistake in the pull request, so it seems. It added previous changes too, eventhough I merged your main branch into mine after you merged the pick-up truck into your main branch. Sorry, I'm kind of new to git :( 

## Related Issues

> ([Github link][autolink-references] to related issues or pull requests)

[autolink-references]: https://help.github.com/articles/autolinked-references-and-urls/

### CODE REVIEW

File: src/vehicle/const.py [hunk 1]
```diff
@@ -41,6 +41,7 @@ class VehicleInterior(str, Enum):
     SEDAN = "sedan"
     STATION_WAGON = "stationwagen"
     PICK_UP_TRUCK = "pick-up truck"
+    NOT_AVAILABLE = "N.v.t."
```
Comment (frenck) [inline src/vehicle/const.py:44]: This is unreachable? `N.v.t.` is filtered by the model

https://github.com/frenck/python-vehicle/blob/main/src/vehicle/models.py#L149

**Review by frenck — CHANGES_REQUESTED** [review]

File: src/vehicle/const.py (hunk 1 above)
Comment (Calimerorulez) [reply src/vehicle/const.py:44]: Good catch. I've been over productive ;). Const.py can indeed be left unchanged. Sorry.

Comment (Calimerorulez) [reply src/vehicle/const.py:44]: I can't seem to delete the const.py file from the pull-request via the site; I'll have to figure out the CLI now. Please bear with me :)

Comment (Calimerorulez) [reply src/vehicle/const.py]: Ok, just removed the added line in const.py.

**Review by frenck — APPROVED** [review]
Thanks, @Calimerorulez 👍

**Pull request merged**